docs/rule/kotlin_library.soy

```diff
@@ -0,0 +1,188 @@
+{namespace kotlin_library}
+
+/***/
+{template .soyweb}
+  {call buck.page}
+    {param title: 'kotlin_library()' /}
+    {param navid: 'rule_kotlin_library' /}
+    {param prettify: true /}
+    {param description}
+      A rule that is used to define a set of Kotlin files that can be compiled
+      together.
+    {/param}
+    {param content}
+
+{call buck.rule}
+{param status: 'FROZEN' /}
+{param overview}
+A <code>kotlin_library()</code> rule is used to define a set of Kotlin
+files that can be compiled together. The main output of a
+{sp}<code>kotlin_library()</code> rule is a single JAR file containing all
+of the compiled class files and resources.
+{/param}
+
+{param args}
+
+{call buck.arg}
+  {param name: 'name' /}
+  {param desc}
+  The name of the rule.
+  {/param}
+{/call}
+
+{call buck.arg}
+  {param name: 'srcs' /}
+  {param default: '[]' /}
+  {param desc}
+  The set of <code>.kt</code> or <code>.kts</code> files to compile for this rule.
+  {/param}
+{/call}
+
+{call buck.arg}
+  {param name: 'resources' /}
+  {param default: '[]' /}
+  {param desc}
+  Static files to include among the compiled <code>.class</code>
+  {sp}files. These files can be loaded via <a
+  href="http://docs.oracle.com/javase/7/docs/api/java/lang/Class.html#getResource(java.lang.String)"
+  target="_blank">Class.getResource()</a>.
+  <p>
+  <strong>Note:</strong> If <code>resources_root</code> isn't set, Buck uses
+  the <code>src_roots</code> property in <a
+  href="{ROOT}concept/buckconfig.html"><code>.buckconfig</code></a> to help
```

Comment:
You should link to the actual src_roots section to make it easier for end-users to find.